### DISCUSSION TITLE
have Event like playnextvideo?

### DISCUSSION THREAD

**Discussion by bookyo** [jhomlala/betterplayer#746] [Q&A]
listen to video player events like:

  initialized,
  play,
  pause,
  seekTo,
  openFullscreen,
  hideFullscreen,
  setVolume,
  progress,
  finished,
  exception,
  controlsVisible,
  controlsHidden,
  setSpeed,
  changedSubtitles,
  changedTrack,
  changedPlayerVisibility,
  changedResolution

have event like playNextVideo?
i use BetterPlayerPlaylist , i want to programmatically change widget style.
how do i know BetterPlayerPlaylist play the next video? once it play next video, i can real-time change the style of the widget in the listview.

**Comment by jhomlala** [comment]
Please update to Better Player 0.0.77. Starting from that version you can use `changedPlaylistItem` event.
